### ISSUE TITLE
The prediction of the pre-trained AASIST

### ISSUE THREAD

**Issue by hkbu-jintian** [clovaai/aasist#17]
Hi, I use the pre-trained weight of AASIST you provided to classify a audio, but always get the opposite result. Like, the test audio is bona-fide, the second score of the output tensor is greater. When the test audio is spoof, the first score of the output tensor is greater. However, according to the paper, the score corresponds to bona-fide and spoof. When a bona-fide audio is given, the first score should be greater, but the result is opposite. I have tested it with several bona-fide audio

<img width="838" height="605" alt="Image" src="https://github.com/user-attachments/assets/13f8b4b1-5585-45d9-bd46-edcc85dc6474" />

```
```
I would like to ask the label match of these two scores when training the model, thank you!
